saskatoon/sitebase/templates/app/list_views/community/data_row.html

```diff
@@ -24,18 +24,11 @@
         </span><br>
         <a href='mailto:{{user.email}}'>{{ user.email }}</a> &nbsp;
         {% if perms.member.change_authuser %}
-            {% if 'core' in user.role_codes or 'pickleader' in user.role_codes %}
-                <a href="{% url 'admin:auth_user_password_change' user.id %}"
+            <a href="{% url 'reset_password' user.id %}"
```

Comment:
we still only want to show the link if the user has a password (i.e. 'pickleader' or 'core', admins can do it themselves)

Reply:
good catch! `has_password` seems to be only id `AuthUserAdmin`, so i added a manual check